### PR TITLE
Ric feat unichem api

### PR DESCRIPTION
**Pull request by ricardoaat**
- Added Unichem Similarity search and Sources fetching API ( glados/api/unichem )
- Added Similarity search and sources pages on chemvue app (Vuejs)

### CODE REVIEW

File: chemvue/src/App.vue [hunk 1]
```diff
@@ -1,132 +1,74 @@
...
-                        <div class="ball ball-4"></div>
-                        <div class="ball ball-5"></div>
-                        <div class="ball ball-6"></div>
-                    </div>
-                    <span class="chembl-logo center-align">ChEMBL</span>
-                    <span class="center-align loading-msg">Loading...</span>
-                </div>
-            </div>
+  <v-app>
+    <EBIHeader></EBIHeader>
```
Comment (nclopezo) [inline chemvue/src/App.vue]: We can start to go towards a standard template for all chembl resources!

File: chemvue/src/components/EBIcomponents/Header.vue [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    // ebiFrameworkPopulateBlackBar();
+  },
+  name: "EBIHeader"
+};
+</script>
+
+<style scoped lang="scss">
+@import "@/assets/sass/ebi/ebi-global.scss";
+
+// @import "@/assets/sass/ebi/ebi-global.scss";
```
Comment (nclopezo) [inline chemvue/src/components/EBIcomponents/Header.vue]: Shouldn't this commented code be removed?

File: chemvue/src/components/shared/Compounds.vue [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      compoundCount: this.compoundsTotal,
+      renderedCompounds: [],
+      page: 1,
+      pages: 0,
+      pageMax: this.maxPerPage,
+      imgBasePath: ""
+    };
+  },
+  created() {
+    console.log("API PATH", process.env.VUE_APP_ROOT_API);
```
Comment (nclopezo) [inline chemvue/src/components/shared/Compounds.vue]: Also, this console.logs should be removed.

File: chemvue/src/components/shared/chembl/Header.vue [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      span {
+        font-size: 30px;
+      }
+    }
+  }
+
+  .search-bar {
+    height: $chembl-header-height;
+    padding: $chembl-header-padding;
+    // float: left;
```
Comment (nclopezo) [inline chemvue/src/components/shared/chembl/Header.vue]: These commented lines can also be removed.

File: src/glados/api/unichem/controllers/similarity.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        init = 0
+        end = 10
+        response = {'inchis': []}
+
+        if not is_int(request.GET['init']) & is_int(request.GET['end']):
+            response['error'] = "Pagination params must be integers"
+
+            return JsonResponse(response)
+
+        if request.GET.get('threshold'):
```
Comment (nclopezo) [inline src/glados/api/unichem/controllers/similarity.py]: Why not
```suggestion
        threshold = request.GET.get('threshold')
        if threshold is None:
          # Fail if threshold was not provided
```

File: src/glados/api/unichem/controllers/similarity.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+            threshold = request.GET['threshold']
+
+        if request.GET.get('init'):
```
Comment (nclopezo) [inline src/glados/api/unichem/controllers/similarity.py]:
```suggestion
        raw_init = request.GET.get('init'):
        if raw_init is not None:
            init = int(raw_init)
        else:
            #Fail 
        
```

File: src/glados/api/unichem/controllers/similarity.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+            response['totalCount'] = total_count
+        else:
+            response['message'] = "No similar compounds"
+            json_response = JsonResponse(response)
+
+            return json_response
+
+        return JsonResponse(response)
+
+    return HttpResponse("Method not supported, SORRY MATE", content_type="application/text")
```
Comment (nclopezo) [inline src/glados/api/unichem/controllers/similarity.py]: Could this be at the beginning of the method?

File: chemvue/src/components/EBIcomponents/Header.vue (hunk 2 above)
Comment (ricardoaat) [reply chemvue/src/components/EBIcomponents/Header.vue]: Yes, deleting this now

**Pull request merged**